### PR TITLE
Tfm upmerge to v.1.3.99 with important fixes for v2.6

### PR DESCRIPTION
**Pull request by ioannisg**
This brings in several fixes. Most important, it formally updates PSA crypto headers, after TFM has switched to Mbedtls 2.26. In addition, it fixes some cache-enabling issues for nRF platforms.

### CODE REVIEW

**Comment by ioannisg** [comment]
Side note: I am going to try upstream the STM downstream patch, so we won't have to keep including it after the upmerges.

File: README.rst [hunk 1]
```diff
@@ -14,18 +14,17 @@ This module in Zephyr has included TF-M and it's dependencies, they are:
...
     repo: https://git.trustedfirmware.org/TF-M/trusted-firmware-m.git
-    commit: c05319dbb991d153081d5bf5314b8a635063b252
+    commit: 9d93424a12b8cc2ba56518f76edadeaabc29e143
     license: BSD-3-Clause
 
 TF-M Tests:
     repo: https://git.trustedfirmware.org/TF-M/tf-m-tests.git
-    tag: TF-Mv1.3.0
-    commit: d2c78e76df1783fc70ec902313daad6f44288d23.
+    commit: eeba9de2f5c19293038d7d551cc0ceffe32d8992..
```
Comment (galak) [inline README.rst:22]: why the `..` at end?

File: README.rst [hunk 2]
```diff
@@ -14,18 +14,17 @@ This module in Zephyr has included TF-M and it's dependencies, they are:
...
     license: Apache 2.0
 
 psa-arch-tests:
     repo: https://github.com/ARM-software/psa-arch-tests
-    commit: b0635d9af8ba031e5d74614b2f0a811993dc122e
+    commit: cd3f3caf3a5bbb543af2206056668422f4e135b9.
```
Comment (galak) [inline README.rst:27]: why `.` at end?

File: init-git.sh [hunk 3]
```diff
@@ -1,8 +1,8 @@
 #!/bin/sh
 
-TFM_GIT_SHA=c05319dbb991d153081d5bf5314b8a635063b252.
-TFM_TESTS_GIT_SHA=d2c78e76df1783fc70ec902313daad6f44288d23.
-PSA_ARCH_TESTS_GIT_SHA=b0635d9af8ba031e5d74614b2f0a811993dc122e
+TFM_GIT_SHA=9d93424a12b8cc2ba56518f76edadeaabc29e143.
+TFM_TESTS_GIT_SHA=eeba9de2f5c19293038d7d551cc0ceffe32d8992..
+PSA_ARCH_TESTS_GIT_SHA=cd3f3caf3a5bbb543af2206056668422f4e135b9.
```
Comment (galak) [inline init-git.sh:5]: what the `.` at end?

**Review by ioannisg — COMMENTED** [review]
Sorry about the dots, @galak, will fix :)

**Comment by ioannisg** [comment]
> This update seems to break support with `lpcxpresso55s69_ns` ... file size is much smaller , and binary doesn't run. Seems to be an upstream problem. Tested with `$ west build -p auto -b lpcxpresso55s69_ns samples/tfm_integration/psa_level_1`. Trying to see what's broken upstream.

Thanks. In the mean time I will test on NRF and STm boards. I 've added the Block:HW test label in the manifest PR.

**Review by anangl — APPROVED** [review]

**Review by galak — CHANGES_REQUESTED** [review]
The tfm repo deleted files that this PR doesn't delete as well.  Please fix this.

**Review by galak — APPROVED** [review]

**Review by microbuilder — APPROVED** [review]
Tested on:

- lpcxpresso5s69_ns
- mps2_an521_nonsecure

**Comment by ioannisg** [comment]
@microbuilder your commit was cherry-picked, can you take a look?

**Pull request merged**